README.md

```diff
@@ -1,15 +1,15 @@
-# [Zen Cart&reg; - The Art of E-Commerce](http://www.zen-cart.com/) 
+# [Zen Cart&reg; - The Art of E-Commerce](https://www.zen-cart.com/) 
 [![Build Status](https://travis-ci.org/zencart/zc-v1-series.svg)](https://travis-ci.org/zencart/zc-v1-series) 
 [![ProjectStatus](http://stillmaintained.com/zencart/zc-v1-series.png)](http://stillmaintained.com/zencart)
-[![Selenium Test Status](https://saucelabs.com/browser-matrix/zencart.svg)](https://saucelabs.com/u/zencart)
+[![Download Zen Cart&reg;](https://img.shields.io/sourceforge/dm/zencart.svg)](http://sourceforge.net/projects/zencart/files/latest/download)
 
 ===============
 
-Zen Cart&reg; v1.5.0 was the first Open Source e-Commerce web application to be fully PA-DSS Certified.
+Zen Cart&reg; v1.5.0 was the first Open Source e-Commerce web application to be fully PA-DSS Certified. The latest stable version is currently v1.5.4 and also carries PA-DSS certification.
 
 Zen Cart&reg; v1.6.0 brings improved templating and responsive design, in addition to increased flexibility for customizing to each storeowner's own preferences.
 
-It's free software, with free support available 24/7 on the Zen Cart&reg; Support Site forums at <http://www.zen-cart.com/forum.php> provided by our enthusiastic community of actual Zen Cart&reg; users, integrators, and the developers themselves.
+It's free software, with free support available 24/7 on the Zen Cart® Support Site forums at <https://www.zen-cart.com/forum.php> provided by our enthusiastic community of actual Zen Cart&reg; users, integrators, and the developers themselves.
```

Comment:
Should we use the html entity or unicode character for displaying the registration mark?